lib/conjur/conjur_config.rb

```diff
@@ -3,13 +3,15 @@
 # from the Conjur application code.
 
 class Conjur::ConjurConfig < Anyway::Config
-  config_name :conjur
+  config_name :config
```

Comment:
This is definitely in the nitpick category, but should these corrections be fixed up into the original commit, rather than changing them here in the spec commit?

Reply:
Yeah probably not worth doing this retroactively.